apps/src/lib/client/eth_bridge_pool.rs

```diff
@@ -67,11 +70,19 @@ pub async fn construct_bridge_pool_proof(args: args::BridgePoolProof) {
 /// Prints out a json payload.
 pub async fn query_bridge_pool(args: args::Query) {
     let client = HttpClient::new(args.ledger_address).unwrap();
-    let response = RPC
+    let response: Vec<PendingTransfer> = RPC
         .shell()
         .read_ethereum_bridge_pool(&client)
         .await
         .unwrap();
-
-    println!("{:#?}", serde_json::to_string_pretty(&response));
+    let pool_contents: HashMap<String, PendingTransfer> = response
+        .into_iter()
+        .map(|transfer| (transfer.keccak256().to_string(), transfer))
+        .collect();
+    if pool_contents.is_empty() {
+        println!("Bridge pool is empty.");
+    } else {
+        println!("Bridge pool contents: ");
```

Comment:
```suggestion
        print!("Bridge pool contents: ");
```
nit: maybe this is the desired output here? `Bridge pool contents: { ...`

Reply:
This I'm not sure about until I see it on the screen. Will get back to it when I have time.

Reply:
It made the whole thing a json payload, which is what I really wanted anyway.